wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -137,6 +142,23 @@ instance
     getNewAddress (accId, passphrase) = do
         clientAddress <- L.newAddress RandomSeed passphrase accId
         decodeCTypeOrFail (cadId clientAddress)
+    -- We are creating a sample HD address here. It has the same
+    -- length (76) as the biggest one created by Daedalus.
+    getFakeChangeAddress = do
+        let passphrase = DBA.replicate 10 255
+        SecretKey sk <- snd <$> keyGen
+        encSK <- mkEncSecret passphrase sk
+        case deriveLvl2KeyPair
+                 (IsBootstrapEraAddr True)
+                 (ShouldCheckPassphrase False)
+                 passphrase
+                 encSK
+                 maxBound
+                 maxBound of
+            Nothing -> throwM $ InternalError "getFakeChangeAddress failed"
```

Comment:
It'd be good to specify which `getFakeChangeAddress` instance has failed exactly.